datafusion/physical-expr/benches/regexp.rs

```diff
@@ -23,15 +23,12 @@ use std::sync::Arc;
 use arrow_array::builder::StringBuilder;
 use arrow_array::{ArrayRef, StringArray};
 use criterion::{black_box, criterion_group, criterion_main, Criterion};
+use datafusion_functions::regex::regexplike::regexp_like;
```

Comment:
Rather than making this dependency, I think we could instead move the benchmark (for regexp_like) to datafusion/functions/benches/regex.rs?

Reply:
I agree that it would be a great idea. Should I port all other regexp benchmark in this PR or in the following one? I will port regexp_like first